### PR TITLE
feat: Key binds screen

### PR DESCRIPTION
**Pull request by gguio**
- [x] test custom binds works
- [x] remove querySelector in favor of React style props, simplify
- [x] in custom keybindings: extract root element from .map into component (eg CustomCommandContainer)
- [x] remove dangerous html
- [x] align ui for custom binds: binds on bottom, actions are grouped (so it feels like binds and custom inputs are in the same container)
- [x] test all works on resolutions (1008x862)
- [x] note text
- [x] fix crashes

### CODE REVIEW

File: src/react/KeybindingsScreenProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+      return [key, newValue]
+    }))
+  }
+
+  if (!commands['custom']) return
+
+  for (const [key, customCommandData] of Object.entries(commands.custom)) {
+    contro.on('pressedKeyOrButtonChanged', (buttonData) => { 
+      customCommandsHandler(buttonData, { customCommandsConfig, ...customCommandData })
```
Comment (zardoy) [inline src/react/KeybindingsScreenProvider.tsx]: Let's keep one event handler, instead can we move the code to existing event handler in contro.ts? And don't forget to merge next

**Review by zardoy — CHANGES_REQUESTED** [review]

File: src/react/KeybindingsScreenProvider.tsx (hunk 1 above)
Comment (gguio) [reply src/react/KeybindingsScreenProvider.tsx]: changed to one handler, but dont see contro.ts file. Is it in src?

**Pull request merged**